auth/src/main/java/com/firebase/ui/auth/data/client/CountryListLoadTask.java

```diff
@@ -38,267 +42,52 @@
         void onLoadComplete(List<CountryInfo> result);
     }
 
-    private static final int MAX_COUNTRIES = 291;
-
     private final Listener mListener;
 
-    public CountryListLoadTask(Listener listener) {
+    List<String> whitelistedCountryCodes;
+    List<String> blacklistedCountryCodes;
+
+    public CountryListLoadTask(Listener listener,
+                               List<String> whitelistedCountryCodes,
+                               List<String> blacklistedCountryCodes) {
         mListener = listener;
+        this.whitelistedCountryCodes = whitelistedCountryCodes;
+        this.blacklistedCountryCodes = blacklistedCountryCodes;
     }
 
     @Override
     protected List<CountryInfo> doInBackground(Void... params) {
-        final List<CountryInfo> countryInfoList = new ArrayList<>(MAX_COUNTRIES);
```

Comment:
@SUPERCILEX I rememeber last time you touched this code you had a reason to keep this list separate from the map in `PhoneNumberUtils`.  Is it not exactly inverted?

Reply:
@lsirac btw holding my approval on this discussion.

Reply:
Ok I went through and actually just printed out the map in each case.  And there is no diff in the map information:
http://www.mergely.com/MhLwmZdA/